torch/onnx/symbolic_opset9.py

```diff
@@ -2318,6 +2318,22 @@ def lstm(g, *args):
         return _lstm_full(g, *args)
 
 
+def lstm_cell(g, *args):
```

Comment:
is number of argument in `args` subject to change? From the symbolic it looks like they are rather static, if so it's better to define them explicitly in function header.

Reply:
The number is fixed, will define explicitly.